lib/matplotlib/axes/_axes.py

```diff
@@ -5923,7 +5923,10 @@ def hist(self, x, bins=None, range=None, normed=False, weights=None,
 
             Default is ``None``
 
-        normed : boolean, optional
+        normed or density : boolean, optional
```

Comment:
suggest putting 'density' first as that will (eventually) be the preferred name.